docs/cmd/kn_container_add.md

```diff
@@ -0,0 +1,61 @@
+## kn container add
+
+Add a container
+
+```
+kn container add NAME
+```
+
+### Examples
+
+```
+
+  The 'container add' represents utility command that prints YAML container spec to standard output. It's useful for
+  multi-container use cases to create definition with help of standard 'kn' option flags. The command can be chained through
+  Unix pipes to create multiple containers at once.
```

Comment:
Maybe mention here that this command supports all container relevant options like `kn service create` ?